src/components/main/AppCard.tsx

```diff
@@ -51,33 +46,46 @@ const AppCard = ({
     id={buildItemFormAppOptionId(name)}
   >
     <StyledCardActionArea>
-      <StyledAppBackground
-        component={image ? 'img' : 'div'}
-        src={image}
-        alt={name}
-      />
-      <CardContent sx={{ width: '100%' }}>
-        <Typography gutterBottom variant="subtitle2" component="div">
-          {name ?? <Skeleton />}
-        </Typography>
-        <Typography
-          variant="body2"
-          color="text.secondary"
+      <Stack direction="row">
+        <Stack
           sx={{
-            display: '-webkit-box',
-            overflow: 'hidden',
-            WebkitBoxOrient: 'vertical',
-            WebkitLineClamp: 3,
+            height: { xs: 40, sm: 70 },
+            width: { xs: 40, sm: 70 },
```

Comment:
You can put these directly on the Stack with the responsive syntax: `height={[40, 70]}`

Reply:
Oh! I didn't know. In the end using a grid was working much better with typography wrapping..